package.json

```diff
@@ -28,6 +28,7 @@
     },
     "dependencies": {
         "chokidar": "^1.7.0",
+        "find": "^0.2.7",
```

Comment:
Yep it does, made the required changes. Didn't notice that glob dit something similar (haven't used the package before).